bpfd/src/bpf.rs

```diff
@@ -38,6 +38,11 @@ impl<'a> BpfManager<'a> {
                 let uuid = entry.file_name().to_string_lossy().parse().unwrap();
                 let mut program = Program::load(uuid)
                     .map_err(|e| BpfdError::Error(format!("cant read program state {}", e)))?;
+                // TODO: State reconstruction not supported for TC programs, so don't add them to the list.
```

Comment:
Do we not need to set the attached state on line 47?

Reply:
I don't think so.  This change avoids inserting the struct into the `programs` list, so I don't think there's any point in setting it.